### ISSUE TITLE
question about loss calculation

### ISSUE THREAD

**Issue by majasomething** [WuJunde/MedSegDiff#35]
I have a question regarding loss calculation:
for training ``loss = (losses["loss"] * weights + losses['loss_cal'] * 10).mean()`` is used.
Why do you weigh the direct prediction of the ground truth higher compared to the comparison with a less noisy version?

Is there a reason that for inference depending on the Dice-score, different composition of cal and sample is used.

Thanks in advance!
